diff --git a/dist/runtime/components/NuxtImg.vue b/dist/runtime/components/NuxtImg.vue
index 4625e1ef0028dd2470ed832106cea08e9adcac38..11685bcb1ba8c6be6daab0deb943c22404d0c1c0 100644
--- a/dist/runtime/components/NuxtImg.vue
+++ b/dist/runtime/components/NuxtImg.vue
@@ -38,7 +38,7 @@ import { useNuxtApp } from '#app/nuxt'
 
 const props = defineProps(imgProps)
 
-const attrs = useAttrs()
+const attrs: any = useAttrs()
 
 const emit = defineEmits<{
   (event: 'load', payload: Event): unknown
